modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -700,17 +706,20 @@ void cv::gimpl::GFluidExecutable::initBufferRois(std::vector<int>& readStarts,
     } // while (!nodesToVisit.empty())
 }
 
-cv::gimpl::GFluidExecutable::GFluidExecutable(const ade::Graph &g,
-                                              const std::vector<ade::NodeHandle> &nodes,
-                                              const std::vector<cv::gapi::own::Rect> &outputRois)
-    : m_g(g), m_gm(m_g)
+cv::gimpl::FluidGraphInputData cv::gimpl::fluidExtractInputDataFromGraph(const ade::Graph &m_g, const std::vector<ade::NodeHandle> &nodes)
 {
+    FluidGraphInputData r;
+    auto& agents_data               = r.m_agents_data;
+    auto& m_scratch_users           = r.m_scratch_users;
+    auto& m_id_map                  = r.m_id_map;
+    auto& m_all_gmat_ids            = r.m_all_gmat_ids;
+    auto& mat_count                 = r.m_mat_count;
```

Comment:
These `m_`s in local names can be misleading, code diff is much smaller with such names though

Reply:
`m_` prefix removed for local variables